### PR TITLE
Add clarification regarding templates in Lovelace cards

### PR DESCRIPTION
**Pull request by eikowagenknecht**
## Proposed change
Small addition to the docs.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/scripts/service-calls.markdown [hunk 1]
```diff
@@ -77,6 +77,8 @@ entity_id: switch.ac
 
 {% endraw %}
 
+Please note that templates can not be used in most standard Lovelace cards (the exception being the Markdown card).
```
Comment (frenck) [inline source/_docs/scripts/service-calls.markdown:80]: This is a page about scripts, I'm not sure why we should add a warning about Lovelace here?

**Comment by eikowagenknecht** [comment]
@frenck I see your point, maybe this is the wrong place.

Since this affects multiple contexts (templating, calls with templating examples, Lovelace cards), the documentation could be:

1. On a page regarding templates (https://www.home-assistant.io/docs/configuration/templating/)
2. On a page regarding Lovelace cards in general (https://www.home-assistant.io/lovelace/cards/)
3. On all the cards pages that don't support templates (all but the Markdown card, e.g. https://www.home-assistant.io/lovelace/alarm-panel/)
4. On all the cards that describe templating use that looks like it would work inside a Lovelace card but actually doesn't (e.g. https://www.home-assistant.io/docs/scripts/service-calls/)

If this behaviour ("templating doesn't work in Lovelace cards") is currently documented somewhere, I couldn't find it.

Thinking of it, I kind of like adding "This card does not support templates." to all cards but the Markdown card, but it would also be quite redundant.

Where would you see this best placed?

**Comment by frenck** [comment]
> If this behaviour ("templating doesn't work in Lovelace cards") is currently documented somewhere, I couldn't find it.

Automations syntax can't be used in Lovelace cards either... I can't use filters like the recorder either in lovelace state cards? I can't use .... on Lovelace either...

My point is, do you want to add this warning to about every page we have that is not about Lovelace?

This page has nothing to do with Lovelace, just like around ~1500 other pages we have... I don't see why it should be added here in this case (or anywhere else for that matter).

**Comment by eikowagenknecht** [comment]
I get that you don't want to include "feature xyz doesn't work here" on some thousand pages when it is obvious to an experienced Home Assistant user.

But I still think a clear description where templates can be used and where they can't be used would benefit the documentation (maybe on the "Templating" site), because for new users it is not quite that obvious as can be seen in questions or misunderstandings about this in the forums.

I'm not familiar enough with Home Assistant (yet) to create a better PR to thoroughly cover all use cases of templating so I'll just close this one now.

**Pull request closed without merging**